next.config.js

```diff
@@ -11,7 +11,16 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
 })
 
 const nextConfig = {
-  inlineImageLimit: 48 // make it tiny so that it doesn't inline,
+  inlineImageLimit: 48, // make it tiny so that it doesn't inline,
+  async redirects() {
+    return [
+      {
+        source: '/old',
```

Comment:
What is `/old`? Also we can do redirects at the server level with a netlify _redirects file if need be. It may make sense to keep them all in one place (all next config or all netlify _redirects)

Reply:
Oh woops, I was just playing with the router trying to learn how redirects work & forgot to delete that.